qdev_wrappers/tests/dataset/test_doNd.py

```diff
@@ -0,0 +1,92 @@
+"""
+These are the basic black box tests for the doNd functions.
+"""
+
+from qdev_wrappers.dataset.doNd import do0d, do1d, do2d
+from qcodes.instrument.parameter import Parameter
+from qcodes import config, new_experiment
+from qcodes.utils import validators
+
+import pytest
+
+config.user.mainfolder = "output"  # set ouput folder for doNd's
+new_experiment("doNd-tests", sample_name="no sample")
+
+_param = Parameter('simple_parameter',
+                   set_cmd=None,
+                   get_cmd=lambda: 1)
+
+_paramComplex = Parameter('simple_complex_parameter',
+                   set_cmd=None,
+                   get_cmd=lambda: 1 + 1j,
+                   vals=validators.ComplexNumbers())
+
+_param_set = Parameter('simple_setter_parameter',
+                       set_cmd=None,
+                       get_cmd=None)
+
```

Comment:
These should be fixtures to make sure that we are independent of the state of the parameters.